content/releases/cloudnative-pg-1-26.0-rc1-released/index.md

```diff
@@ -0,0 +1,164 @@
+A new resource, `BarmanObjectStore`, will replace the `.spec.barmanObjectStore`
+stanza, incorporating the `.spec.backup.retentionPolicy` option.
+
+In CloudNativePG 1.28, Barman Cloud will be fully removed from CloudNativePG’s
+core, making the plugin the only community-supported method for continuous
+backup on object stores. You have until then to complete your migration.
+
+This marks a significant milestone in CloudNativePG’s evolution—the culmination
+of a multi-year effort that introduced CNPG-I, our extensible plugin interface.
+It is a crucial step toward making CloudNativePG a backup-agnostic solution
+while also enabling leaner operand images by removing the need to bundle Barman
+Cloud directly.
+
+Additionally, in version 1.26, the `hibernate` command of the plugin now
+leverages the declarative hibernation capability. Instead of executing an
+imperative hibernation process that destroyed replica PVCs, it will now
+annotate the cluster, aligning with CloudNativePG’s declarative approach.
+
+## Get Involved and Make a Difference!
+
+This release candidate offers a unique opportunity to test the new features in your
+environment before the final release. Although **not recommended for production
+use**, simulating your workloads can help uncover potential issues and validate
+feature stability.
+
+Your feedback is vital to ensuring that CloudNativePG 1.26 maintains its
+reputation as the premier Kubernetes operator for PostgreSQL.
+
+[Learn more about testing and contributing](https://cloudnative-pg.io/documentation/preview)!
+
+## There's More...
+
+Explore other improvements in this release, including:
+
+- A new annotation to enable/disable webhook validation.
+- A configuration option to set the TCP timeout for replicas, improving
+  failover recovery speed.
+- Integration with autoscalers like Karpenter for better node drain management.
+
+Dive into the full details in the
+[release notes for 1.26 RC1](https://cloudnative-pg.io/documentation/preview/release_notes/v1.26/).
+
+## Testing
+
+The stability of each CloudNativePG release relies on the community’s
+engagement. Testing your workloads with this release candidate helps identify
+bugs and regressions early.
+
+- View the [open issues for the 1.26 release](https://github.com/cloudnative-pg/cloudnative-pg/milestone/25).
+- Report bugs directly on [GitHub](https://github.com/cloudnative-pg/cloudnative-pg/issues/new/choose).
+
+## Release Timeline
+
+CloudNativePG 1.26 RC1 is the first in a series of release candidates.
+Additional RCs may follow as needed before the final release, currently planned
+in the second half of April 2025.
+
+## Join the Community
+
+Join our vibrant, open-source, vendor-neutral community! [Connect with us](https://github.com/cloudnative-pg/cloudnative-pg?tab=readme-ov-file#communications)!
+
+Thank you for your continued support of CloudNativePG. Your contributions help
+us advance the Kubernetes-native PostgreSQL experience!
+
```

Comment:
what's with all the commented out stuff below?

Reply:
That's for PostgreSQL.org news